src/main/java/org/mockito/exceptions/stacktrace/StackTraceCleaner.java

```diff
@@ -17,10 +17,10 @@
 public interface StackTraceCleaner {
 
     /**
-     * Decides if element is excluded.
+     * Decides if element is included.
      *
      * @param candidate element of the actual stack trace
      * @return whether the element should be excluded from cleaned stack trace.
      */
-    boolean isOut(StackTraceElement candidate);
+    boolean isIn(StackTraceElement candidate);
```

Comment:
Was implicated in a broken build at my end. The code that was causing it was suffering from a double-negative, so I refactored this into the positive.